scripts/setup.js

```diff
@@ -116,6 +116,8 @@ shell.cd('../strapi-upload-local');
 watcher('📦  Linking strapi-upload-local...', 'npm link --no-optional', false);
 shell.cd('../strapi-upload-aws-s3');
 watcher('📦  Linking strapi-upload-aws-s3...', 'npm link --no-optional', false);
+shell.cd('../strapi-upload-rackspace');
+watcher('📦  Linking strapi-upload-rackspace...', 'npm link --no-optional', false);
```

Comment:
To improve the setup time, can you remove the link by default of `strapi-upload-rackspace`.
And can you remove the link of `strapi-upload-aws-s3` too please.

Every one will not use all external providers, so we haven't to link all of them by default.